packages/client/components/DashNavList/EmptyTeams.tsx

```diff
@@ -19,15 +19,17 @@ const EmptyTeams = (props: Props) => {
       fragment EmptyTeams_organization on Organization {
         id
         name
-        publicTeams {
+        teams {
           id
+          isViewerOnTeam
           ...PublicTeamsModal_team
         }
       }
     `,
     organizationRef
   )
-  const {id: orgId, publicTeams} = organization
+  const {id: orgId, teams} = organization
+  const publicTeams = teams.filter((team) => !team.isViewerOnTeam)
```

Comment:
+1 Is this really how we want to handle public teams? When testing on the Parabol org I found it quite confusing. As admin it appears we have plenty of public teams, but in reality there were none.
I could imagine an org admin falling into the same trap.